src/Illuminate/Support/Collection.php

```diff
@@ -1116,6 +1116,21 @@ public function take($limit)
     }
 
     /**
+     * Drop the first {$limit} items from the collection.
+     *
+     * @param  int  $limit
+     * @return  static
+     */
+    public function drop($limit)
+    {
+        if ($limit < 0) {
+            $limit = abs($limit);
```

Comment:
> I was mostly trying to mimic the behavior of take.

But you've not minimed the behaviour of take. Negative values in take, take from the back. You've just ignored the sign of the limit here instead.

> a negative limit should be? Dropping from the end?

Either that, or just throwing an error if it's negative. Dropping from the end would probably be better.